frontend-react/src/components/ManagePublicKey/ManagePublicKeyChooseSender.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from "react";
+import { Form, Dropdown, Button, FormGroup } from "@trussworks/react-uswds";
+
+import { UseOrganizationSenders } from "../../hooks/UseOrganizationSenders";
+import Spinner from "../Spinner";
+
+type Props = {
```

Comment:
Let's name prop types after their components, which helps to clarify if we need to import them elsewhere: `ManagePublicKeyChooseSenderProps`.